src/native/libs/System.Native.Browser/libSystem.Native.Browser.extpost.js

```diff
@@ -4,10 +4,13 @@
 
 /* eslint-disable no-undef */
 /* eslint-disable space-before-function-paren */
+/* eslint-disable @typescript-eslint/no-unused-vars */
 var fetch = fetch || undefined; var dotnetNativeModuleLoaded = false; var dotnetInternals = null;
 export function dotnetInitializeModule(internals) {
     if (dotnetNativeModuleLoaded) throw new Error("Native module already loaded");
-    dotnetInternals = internals;
     dotnetNativeModuleLoaded = true;
-    return createDotnetRuntime(dotnetInternals[0/*InternalExchangeIndex.RuntimeAPI*/].Module);
+    if (!Array.isArray(internals)) throw new Error("Expected internals to be an array");
+    dotnetInternals = internals;
+    const runtimeApi = internals[0] = internals[0/*InternalExchangeIndex.RuntimeAPI*/] || {};
+    return createDotnetRuntime(runtimeApi.Module || {});
```

Comment:
Is the `|| {}` needed if line 14 is updated to be `|| { Module: { } }`?

I'd prefer to have some API, perhaps `createDefault RuntimeApi()`, that handles the default case.